examples/server/server.cpp

```diff
@@ -2035,20 +2053,20 @@ struct server_context {
                 n_batch /= 2;
                 i -= n_batch;
 
-                continue;
+                continue; // continue loop of n_batch
```

Comment:
In fact, the loop is to retry with llama_decode with smaller n_batch, until it successful.

For now I don't know how to rephrase the idea, so I just leave it here to distinguish it from the continue and break statements below.

Or maybe we don't need these comments at all, do you have any input for that @ggerganov ?

Reply:
I decide to keep the comment this way, because the goal is to reflect the variable name, which helps when we navigate in a nested loop (but not to explain what the code does).

If you're ok with that, then I'll merge this PR @phymbert